__tests__/api/update_profile.js

```diff
@@ -1,15 +1,31 @@
 const frisby = require('frisby');
+const joi = frisby.joi
 
-it ('POST should return a status of 201 Created', function () {
+frisby.globalSetup({
```

Comment:
Incorrect Flow, you should be getting the token from the login endpoint. So I can test automatically.